refinery/tests/mod_migrations/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [features]
 sqlite = ["barrel/sqlite3"]
-postgresql = ["barrel/pg"]
+postgres = ["barrel/pg"]
```

Comment:
I just noticed this and I'm wondering if we just wanna expose this via `pg` as well. It is pretty short and snappy and everybody should know what is meant

Reply:
yeah makes sense there, updated it! thanks 😃
Or are you mean globally? i used postgres because it's the name of the driver, as with rusqlite and mysql due to the traits being driver specific